CHANGELOG.md

```diff
@@ -34,9 +34,11 @@
 - Added units test for sprints ([#224](https://github.com/chingu-x/chingu-dashboard-be/pull/224))
 
 ### Changed
+
 - Updated cors origin list ([#218](https://github.com/chingu-x/chingu-dashboard-be/pull/218))
 - refactored unit tests for the ideations controller and services([#219](https://github.com/chingu-x/chingu-dashboard-be/pull/219))
 - revised tech selections route to update only one tech per request([#221](https://github.com/chingu-x/chingu-dashboard-be/pull/221))
+- Updated readme for installation part ([#225])(https://github.com/chingu-x/chingu-dashboard-be/pull/225)
```

Comment:
THis one needs to be moved to the unrelease section now, after the latest release merge. 🙏